test/test_runs.py

```diff
@@ -37,6 +38,20 @@
 NAME2 = 'grape_test2'
 
 
+def debug(msg: str, level: int = 1):
+    '''
+    Print a debug message with context.
+
+    Args:
+        msg: The message string.
+        level: The stack level. Default is the caller.
+    '''
+    lnum = inspect.stack()[level].lineno
+    fname = inspect.stack()[level].filename
+    base = os.path.basename(fname)
+    print(f'\x1b[35mDEBUG:{base}:{lnum}: {msg}\x1b[0m\n')
```

Comment:
Perhaps [Colorama](https://pypi.org/project/colorama/)?

Reply:
thank you, the naming should be consistent.

Reply:
To be fair i never considered windows. Given that, there other places that would need to be updated.
Would you be okay with allowing this?
Then we can create a ticket to update coloring everywhere.

Reply:
100%